spec/system/assessor_interface/completing_assessment_spec.rb

```diff
@@ -410,6 +416,31 @@ def then_the_application_form_is_declined
     )
   end
 
+  def and_i_click_view_timeline
+    assessor_application_page.click_link(
+      "View timeline of this applications actions",
+    )
+  end
+
+  def then_i_see_application_declined
+    timeline_item = assessor_timeline_page.timeline_items.first
+
+    unless timeline_item.title.has_content?("Email sent")
```

Comment:
Rather than checking the title of the first timeline event, we should make sure that a timeline event exists with that title regardless of whether it's the first one, by searching through the array of `timeline_events`.